`test (vitest)`: Disable vitest globals (#1387)
seek-oss/sku: docs/docs/vite.md

}
```

**Globals disabled**

> Jest has their globals API enabled by default. Vitest does not. Sku adopts the Vitest defaults, meaning you will need to explicitly import test functions (`expect`, `beforeEach`, etc.). This change results in a cleaner global namespace, better type safety, clearer dependencies and is more consistency with modern javascript practices.

Reviewer: ```suggestion
Jest enables global APIs such as `it`, `describe`, `beforeAll`, etc., by default. Vitest does not. Sku adopts the Vitest defaults, meaning you will need to explicitly import test functions (`expect`, `beforeEach`, etc.). This change results in a cleaner global namespace, better type safety, clearer dependencies and is more consistent with modern JavaScript practices.
```

```diff
+ import { describe, expect, it } from 'vitest';
```

Be aware that since globals are disabled, some common libraries like `testing-library` will not run auto DOM cleanup. If using these libraries, you will need to add cleanup to your configured `setupTests` file.

```diff
import '@testing-library/jest-dom';

+ import { cleanup } from '@testing-library/react';
+ import { afterEach } from 'vitest';

> + afterEach(() => {
> +  cleanup();
> + });

Reviewer: I like how concise this is but happy to leave it as is.
```suggestion
+ afterEach(cleanup);
```
